api/nodecontrol/client.go

```diff
@@ -44,17 +47,29 @@ func (c *ControlAPIClient) Auction(tags []string) (*nodegen.AuctionResponseJson,
 	return resp, nil
 }
 
-func (c *ControlAPIClient) Ping() (*nodegen.NodePingResponseJson, error) {
-	msg, err := c.nc.Request(actors.PingSubject(), nil, DefaultRequestTimeout)
+func (c *ControlAPIClient) Ping() ([]*nodegen.NodePingResponseJson, error) {
+	resp := []*nodegen.NodePingResponseJson{}
+	pingRespInbox := nats.NewInbox()
+
+	_, err := c.nc.Subscribe(pingRespInbox, func(m *nats.Msg) {
+		envelope := new(actors.Envelope[nodegen.NodePingResponseJson])
```

Comment:
I know this might seem minor, but we should not be using the actors envelope here. There should be an envelope in the control API package. One is internal and used for actors alone while the other is part of the public API.

Reply:
I'm not even going to make a copilot joke here. Nope.